src/arch/InputHandler/InputHandler_DirectInput.cpp

```diff
@@ -411,11 +398,8 @@ InputHandler_DInput::UpdatePolled(
 							pos = JOY_AUX_4;
 							val = state.rglSlider[1];
 						} else
-							LOG->MapLog("unknown input",
-										"Controller '%s' is returning an "
-										"unknown joystick offset, %i",
-										device.m_sName.c_str(),
-										in.ofs);
+							Locator::getLogger()->trace("Controller '{}' is returning an unknown joystick offset, {}",
```

Comment:
We want to be careful converting MapLogs to traces due to their possibly high traffic usage. 
In this specific case, it might be better to make this a warn. If it crops up as an issue later, we could remove this log entirely or find a new solution.

Reply:
Fixed.